src/client/components/layout.js

```diff
@@ -145,25 +144,25 @@ class Layout extends React.Component {
 				}
 				{!(homepage || hideSearch) &&
 				<Navbar.Form pullRight
-					action="/search"
-					role="search"
+					 action="/search"
+					 role="search"
 				>
-					<FormGroup>
+					<div className="form-group">
```

Comment:
Should've added a comment here. The usage of the react-bootstrap FormGroup component here was causing a DOM mutation.